laws/bifunctor.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const {identity, compose} = require('fantasy-combinators');
+const {bimap} = require('..');
+
+/**
+### Bifunctor
+
+1. `p.bimap(a =>, b => b)` is equivalent to `p` (identity)
+2. `p.bimap(compose(f1)(f2), compose(g1)(g2))` is equivalent to `p.bimap(f1, g1).bimap(f2, g2)` (composition)
+
+**/
+
+const identityʹ = t => eq => x => {
```

Comment:
I believe it has been possible since ES5, as it belongs to the [`Modifier letter (Lm)`](https://codepoints.net/U+02B9) category.